### DISCUSSION TITLE
pip install manimgl fails

### DISCUSSION THREAD

**Discussion by nandudb** [3b1b/manim#1413] [General]
I see the below error when i install `pip install manimgl`

I checked my python version 
`$which python
    python: aliased to /usr/local/bin/python3
  $python --version
    Python 3.9.2`

`   ERROR: Command errored out with exit status 1:
     command: /System/Library/Frameworks/Python.framework/Versions/2.7/Resources/Python.app/Contents/MacOS/Python -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/private/var/folders/gm/06_cgzys2xv0w4xjzyd3g64h0000gn/T/pip-install-zESjRk/moderngl/setup.py'"'"'; __file__='"'"'/private/var/folders/gm/06_cgzys2xv0w4xjzyd3g64h0000gn/T/pip-install-zESjRk/moderngl/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /private/var/folders/gm/06_cgzys2xv0w4xjzyd3g64h0000gn/T/pip-install-zESjRk/moderngl/pip-egg-info
         cwd: /private/var/folders/gm/06_cgzys2xv0w4xjzyd3g64h0000gn/T/pip-install-zESjRk/moderngl/
    Complete output (5 lines):
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/private/var/folders/gm/06_cgzys2xv0w4xjzyd3g64h0000gn/T/pip-install-zESjRk/moderngl/setup.py", line 13, in <module>
        raise Exception('Python 2 is not supported!')
    Exception: Python 2 is not supported!
    ----------------------------------------
ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.`

**Comment by naveen521kk** [comment]
For some reason python2 is used as specified in the logs. Can you use `python3` command?

**Reply by kilacoda-old** [reply]
They've aliased `python` to `python3` apparently, which is probably confusing the OS which one to use.

**Reply by naveen521kk** [reply]
or rather let them use `python3.9` command.
